src/deepsparse/loggers/helpers.py

```diff
@@ -80,7 +82,10 @@ def unwrap_logs_dictionary(
                 if parent_identifier
                 else child_identifier
             )
-            if isinstance(child_value, dict):
+            if isinstance(child_value, list):
```

Comment:
do we always generically want to unwrap lists? Should only be if it's a `BatchResult`, no?